pwndbg/color/__init__.py

```diff
@@ -55,8 +56,14 @@ def colorize(x, color): return color + terminateWith(str(x), color) + NORMAL
 
 @pwndbg.memoize.reset_on_stop
 def generateColorFunctionInner(old, new):
-    def wrapper(text):
-        return new(old(text))
+    # End to end tests check for whole output, we might not want to deal with colors there
+    if os.environ.get('PWNDBG_TESTS_DISABLE_COLORS') != 'yes':
```

Comment:
This should be checked once at boot-up, rather than every time any color function is invoked.

Separately, I don't see a reason for this to not be a `config` setting.  We can pull the default value from the environment.

Reply:
I am not sure if we can omit the condition here if we want to have it in config variable, but I will give it a try.